packages/core/parcel/package.json

```diff
@@ -14,7 +14,7 @@
     "type": "git",
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
-  "bin": "src/bin.js",
+  "bin": "lib/bin.js",
```

Comment:
```suggestion
  "bin": "src/bin.js",
```

Reply:
Not sure how this happened... I never touched this file. Thanks for checking it

Reply:
Probably something like this:
```
yarn build
git commit -a
```